dimagi/utils/memory/utils.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import print_function
+from sys import getsizeof, stderr
+from itertools import chain
+from collections import deque
+try:
+    from reprlib import repr
+except ImportError:
+    pass
+
+
+# From: https://code.activestate.com/recipes/577504/
+def total_size(o, handlers={}, verbose=False):
```

Comment:
Ah. Nice catch. (this was written by a python core dev: https://twitter.com/raymondh, who I guess are fallible too...)